examples/data-objects/webflow/src/test/layout.spec.ts

```diff
@@ -58,8 +60,11 @@ describeCompat("Layout", "LoaderCompat", (getTestObjectProvider) => {
 
 	afterEach(() => {
 		layout.remove();
-		layout = undefined;
-		root = undefined;
+
+		// Out of paranoia, force setting to 'undefined' to ensure that 'layout' and 'root'
+		// are re-created between tests.
+		layout = undefined as any;
+		root = undefined as any;
```

Comment:
Semantically, the test is replace 'layout' and 'root' before each test.

Forcibly setting these values to undefined after each test is an unnecessary/paranoid way to ensure this happens.

Alternatives to casting to any would be to remove these lines or sprinkle '!' or checks for undefined throughout the tests.